app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -95,6 +95,10 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         super.onCreate(savedInstanceState)
         AndroidSupportInjection.inject(this)
 
+        //push all content up when keyboard comes
+        activity!!.window.setSoftInputMode(
+                WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN)
+
```

Comment:
By doing this you're hiding the toolbar, which is not exactly what we're looking for. :(